player/src/HttpMp4Video.tsx

```diff
@@ -95,6 +95,67 @@ export const HttpMp4Video: React.FC<HttpMp4VideoProps> = ({
 
   const __sensorTmRef = useRef<TransformationMatrix>()
 
+  /**
+   * Show debug logs with information received from
+   * 'progress' & 'timeupdate' events including the current
+   * up time, delay and end time of last buffer.
+   */
+  useEffect(() => {
+    const debugKey = localStorage.getItem('debug')
+    if (
+      // Don't initiate event listeners if debug is not on
+      debugKey !== 'msp:http-mp4-video' ||
```

Comment:
`debug` also supports `msp:*` for example, but that's not reflected here.

Reply:
Done